.storybook/main.js

```diff
@@ -32,6 +32,12 @@ module.exports = {
       ],
       include: path.resolve(__dirname, "../"),
     });
+    config.module.rules.push({
```

Comment:
What is that for?

Reply:
To solve this: 
```
ERROR in ./node_modules/lnmessage/dist/index.js 4:0-50
Module not found: Error: Can't resolve './crypto' in '/Users/kwinten/Alby/lightning-browser-extension/node_modules/lnmessage/dist'
Did you mean 'crypto.js'?
BREAKING CHANGE: The request './crypto' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

I don't understand the details, maybe @escapedcat remembers.

Reply:
hmm... in alby-tools this worked fine: https://github.com/getAlby/alby-tools/blob/master/package.json#L5

also is storybook even using our build system? if the build system can not handle it then we would need some webpack config update I guess?

@kiwiidb where do you see that error that you posted? Can try to remove this and run it again?

Reply:
oh I missed it's in the webpack config also. 
but somewhat I would like to get rid of that and the npm package should be in a way that it also works for us - because I think we don't have such a strange build system. 
maybe we can make a PR there?

Reply:
Background: https://webpack.js.org/configuration/module/#resolvefullyspecified  

Removing `"type": "module",` from the "lnmessage" `package.json` already makes it work without this special webpack option.  
`"type": "module",` is letting node know that this is an [esm package](https://nodejs.org/api/esm.html#introduction) but this shouldn't be needed for a package that should run in the browser (only).  

I can create a PR for this and ask for feedback.

Reply:
what's with the following package.json keys:
`source`, `main`, `unpkg`, `exports` ? 

packages that can run in different environments should publish the package valid for the different environments. 

Does the build step need to be improved? https://github.com/aaronbarnardsound/lnmessage/blob/master/package.json#L14 
`tsc` is not enough?

I don't know enough about the build setups in 2022, but I used `microbundle` recently in this package: https://github.com/getAlby/alby-tools/blob/master/package.json#L28

Reply:
We started a discussion with Aaron on this and will solve this later on.

Reply:
I think this can be removed no. 
Actually if not 1000% needed I would like to resolve this now. because otherwise we will forget it and not take care of it and it will stay in there.